zio-config/src/main/scala/zio/config/actions/Read.scala

```diff
@@ -13,6 +13,8 @@ object Read {
       report: Ref[ConfigReport]
     ): ZIO[ConfigSource, ReadErrors, (ConfigReport, B)] =
       configuration match {
+        case Config.Pure(value) => report.get.map(t => (t, value))
```

Comment:
I remember in discussion, we don't want to use the word `Pure`. I couldn't come up with an alternative name though.